src/merkleDistributor/model/Distribution.ts

```diff
@@ -0,0 +1,23 @@
+export type DistributionRecipients = {
+  [address: string]: Pick<DistributionRecipient, "amount" | "accountIndex" | "metadata">;
+};
+export type DistributionRecipientsWithProofs = { [address: string]: DistributionRecipient };
+
+export type DistributionRecipient = {
+  amount: string;
+  accountIndex: number;
+  windowIndex: number;
```

Comment:
Can you explain why these are needed? I struggle to wrap my around how these indices are used 🤔

Reply:
`windowIndex` is used to allow multiple distributions (airdrops) using the same contract without the need of deploying other MerkleDistributor contracts. 
`accountIndex` is an index used to identify the account in the smart contract, instead of using the account address. The contract has a map [windowIndex][accountId][boolean] which stores if the account claimed their tokens or not